pkg/operations/remove.go

```diff
@@ -78,26 +75,34 @@ func RemoveVMContainer(result *runtime.ContainerInspectResult) {
 }
 
 // StopVM stops or kills a VM
```

Comment:
Updated comment: `StopVM removes networking of the given VM and stops or kills it`